paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -837,30 +838,70 @@ class PaymentSheet internal constructor(
          * The border color of the primary button.
          */
         @ColorInt
-        val border: Int
+        val border: Int,
+        /**
+         * The background color for the primary button when in a success state. If null,
+         * defaults to base green background color.
```

Comment:
Since you can’t actually pass `null`, we should just remove the second sentence.